
perl 5.26 dropped '.' from the module search path (@INC), so to allow
5.34 to find build-time code in inc, add '.' to the end of the path
in INC.

diff -ur Net-SSLeay-1.80.orig/Makefile.PL Net-SSLeay-1.80/Makefile.PL
--- Net-SSLeay-1.80.orig/Makefile.PL	2016-07-27 17:55:21.000000000 +0000
+++ Net-SSLeay-1.80/Makefile.PL	2022-01-02 23:43:45.360872633 +0000
@@ -1,5 +1,10 @@
 #!/usr/bin/perl
 
+# allow perl 5.34 to find modules relative to .
+BEGIN {
+	push @INC, '.';
+}
+
 use strict;
 use warnings;
 use lib 'ext';
